site/assets/scss/_toc.scss

```diff
@@ -16,23 +16,34 @@
     ul {
       padding-left: 0;
       margin-bottom: 0;
+      margin-left: -.75rem;
       list-style: none;
 
       ul {
         padding-left: 1rem;
-        margin-top: .25rem;
-      }
-    }
+        margin-left: -1rem;
 
-    li {
-      margin-bottom: .25rem;
+        a {
+          padding-left: 1.375rem;
+        }
+      }
     }
 
     a {
+      display: block;
+      padding: .125rem 0 .125rem .625rem;
       color: inherit;
+      text-decoration: none;
+      border-left: .125rem solid transparent;
+
+      &:hover,
+      &.active {
+        color: var(--bd-violet-bg);
```

Comment:
Contrast ratio is not that good in dark mode. Maybe we could use a lighter violet or a whitish color.

![Screenshot 2022-12-29 at 20 34 46](https://user-images.githubusercontent.com/17381666/210003797-15fddff6-8eba-4e10-b590-d2684980e52c.png)

Reply:
IDK if it's enough but it's way better with https://github.com/twbs/bootstrap/pull/33428/commits/03b552810f936f9a1e55f93e4133d419e4854332 and the 3.9:1 contrast ratio :) Thx @mdo